catalogue/webapp/pages/works.tsx

```diff
@@ -355,16 +340,20 @@ const Works = ({
   );
 };
 
-export const getServerSideProps = async (
-  ctx: Context
-): Promise<{ props: Props }> => {
-  const globalContextData = getGlobalContextData(ctx);
-  const params = WorksRoute.fromQuery(ctx.query);
-  const { enableColorFiltering } = ctx.query.toggles;
-  const _queryType = cookies(ctx)._queryType;
+export const getServerSideProps: GetServerSideProps<
+  Props | AppErrorProps
+> = async context => {
+  const globalContextData = getGlobalContextData(context);
+  const params = WorksRoute.fromQuery(context.query);
+  // TODO: Fix this, we should have the toggles typesd at this point
+  const { enableColorFiltering } = (context.query.toggles as unknown) as Record<
```

Comment:
would be ideal if we had the type for the togglesContext

Reply:
The trouble is propagating that time down the pipes.

As we've hacked it a little and attached the cached values onto `context.query`, which it self has a type of `{ [key: string]?: string | string[] }` as it's representing the querystring type from nodes http type system.

It's a part of the system I have really tried to work out... although now that we're speaking about it, [I think I know the answer for this case](https://github.com/wellcomecollection/wellcomecollection.org/pull/5819/commits/8713de1cbf8de1d8a81889eede4a952e63908550).